patches/chrome-android-java-src-org-chromium-chrome-browser-night_mode-GlobalNightModeStateProviderHolder.java.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/night_mode/GlobalNightModeStateProviderHolder.java b/chrome/android/java/src/org/chromium/chrome/browser/night_mode/GlobalNightModeStateProviderHolder.java
+index 87a0bad6c8494b0b61e65bfa4ae81d88bc076dd2..43b71a01b9ef7087454df3ddf78bd0f9476ab28d 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/night_mode/GlobalNightModeStateProviderHolder.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/night_mode/GlobalNightModeStateProviderHolder.java
+@@ -66,6 +66,7 @@ public class GlobalNightModeStateProviderHolder {
+                         PowerSavingModeMonitor.getInstance(),
+                         ChromePreferenceManager.getInstance());
+             }
++            sInstance = new GlobalNightModeStateController(SystemNightModeMonitor.getInstance(), PowerSavingModeMonitor.getInstance(), ChromePreferenceManager.getInstance());
```

Comment:
This is porting the change we have in browser-android-tabs https://github.com/brave/browser-android-tabs/pull/2071/files?file-filters%5B%5D=.java#diff-bd6d2197b6057142be63f500a98ab944R60

Reply:
cc @bridiver ^^